### ISSUE TITLE
Repo stats API out-dated

### ISSUE THREAD

**Issue by dsquareindia** [PyGithub/PyGithub#443]
Hi, I'm encountering a small issue wherein the `.contributions` attribute of a `NamedUser` instance returns `None`. Here's the code:

``` python
user = gh.get_user('example_user')
print user.contributions
```

The output is always `None`. Here `gh` is the `MainClass.Github` authorized instance. Am I doing something wrong here? Thanks a lot for the help!
P.S. My `pygithub` is in sync with upstream.

**Comment by kinmanz** [comment]
I think github api just doesn't provide function to get contributions for certain user, I don't know why authors of this library place such field to User class. You can look up here : https://developer.github.com/v3/repos/statistics/#get-the-number-of-additions-and-deletions-per-week

**Comment by sfdye** [comment]
Looking at the latest API:
https://developer.github.com/v3/repos/statistics/#get-contributors-list-with-additions-deletions-and-commit-counts

I believe the `contributions` is an out-dated attribute from the old API:
`/repos/:owner/:repo/contributors`

This attribute should be removed and this API needs to be updated as well.

**Comment by s-t-e-v-e-n-k** [comment]
The statistics API was implemented years ago -- first added in a4910ac922c12cd1e3ce58766ebda8780391aae4 and completed in c6671c64668d67018cc89b364887dd056a498277.

(The relevant methods are Repository.get_stats_*)
